tempodb/compactor_integration_test.go

```diff
@@ -0,0 +1,393 @@
+package tempodb
```

Comment:
Do we need to add a go build flag here to limit the scope of the build?  This will require that the jobs run the job with the given tags to include.

Reply:
Applies to the integration test files.

Reply:
I considered build tags, but I think we'd need to add tags to every single test file to split the tests in CI. The trick here is we are using a prefix on the test name. The make file runs all tests that start with `TestIntegration` in a different target than just `Test`